src/alist_naudio.c

```diff
@@ -263,6 +263,16 @@ static void MP3(struct hle_t* hle, uint32_t w1, uint32_t w2)
     mp3_task(hle, index, address);
 }
 
+static void OVERLOAD(struct hle_t* hle, uint32_t w1, uint32_t w2)
+{
+    /* Overload distortion effect for Conker's Bad Fur Day */
+    uint16_t dmem = (w1 & 0xfff) + NAUDIO_MAIN;
+    int16_t gain = w2 & 0x7fff;
```

Comment:
```suggestion
    int16_t gain = (int16_t)(uint16_t)w2;
```

The ucode uses mtc2 to put the gain in vector register, there is no masking taking place as far as I can see (but I may have missed something, it's been a long time I haven't touched any RSP assembly...)